src/Console/Cache.php

```diff
@@ -59,11 +71,13 @@ protected function getHelp()
 		return $help;
 	}
 
-	public function __construct(App\Mode $appMode, array $argv = null)
+	public function __construct(App\Mode $appMode, Configuration $config, ICacheDriver $cache, array $argv = null)
```

Comment:
If you only need the cache driver name from the config, why not have it as an argument instead of requiring the whole config object?

Reply:
Done, now every (string)$cache returns the driver name ;-)